Miscellaneous/docker/full_stack_example/docker-compose-windows.yml

```diff
@@ -0,0 +1,174 @@
+---
+version: "2.1"
+services:
```

Comment:
Is there much difference between the OSX, Windows, and Linux versions? Could we generate them from a template to make this more maintainable over time?